auth/tests/unit/utils_test.py

```diff
@@ -0,0 +1,15 @@
+import gcloud.aio.auth.utils as utils  # pylint: disable=unused-import
+
+
+def test_importable():
```

Comment:
Feel free to nuke this importable test -- we've just been using that as a placeholder for files without actual tests.

Reply:
Will do